src/DIRAC/Resources/Computing/HTCondorCEComputingElement.py

```diff
@@ -443,77 +404,56 @@ def getJobStatus(self, jobIDList):
         resultDict = {}
         condorIDs = {}
         # Get all condorIDs so we can just call condor_q and condor_history once
-        for jobRef in jobIDList:
-            job, _, jobID = condorIDAndPathToResultFromJobRef(jobRef)
-            condorIDs[job] = jobID
+        for diracJobID in jobIDList:
```

Comment:
```suggestion
        condorIDs = dict({(pR := jobRef.split(":::")[0], self._DiracToCondorID(pR)) for jobRef in jobIDList})
```
Fixed to make a dict.

Reply:
I did not implement this: it creates a set of tuples.
But I shorten the number of lines in the loop if it helps.

Reply:
I wanted to propose something, where we don't have to say diracJobID.
I fixed the comprehension to make a dict at the end.